swap_meet/vendor.py

```diff
@@ -0,0 +1,90 @@
+from .item import Item
+import operator
+
+class Vendor:
+
+
+    def __init__(self, inventory = None):
+        '''constructs instance of Vendor, 
+        initializing inventory with default None'''
+        
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self, item):
+        '''receives item as an argument and adds it to Vendor inventory'''
+        
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        '''receives item as an argument and removes it from Vendor inventory'''
+        
+        if item in self.inventory:
```

Comment:
I found this fancy way to one-line if the item is in the inventory then remove it: https://stackoverflow.com/questions/4915920/how-to-delete-an-item-in-a-list-if-it-exists/4915964

This is an example of a ternary operator: https://book.pythontips.com/en/latest/ternary_operators.html